pkg/controller/controller.go

```diff
@@ -157,10 +157,12 @@ const (
 func (c *controller) reconcileBroker(broker *v1alpha1.Broker) {
 	glog.V(4).Infof("Processing Broker %v", broker.Name)
 
-	username, password, err := GetAuthCredentialsFromBroker(c.kubeClient, broker)
+	username, password, err := getAuthCredentialsFromBroker(c.kubeClient, broker)
 	if err != nil {
-		glog.Errorf("Error getting broker auth credentials for broker %v: %v", broker.Name, err)
-		c.updateBrokerReadyCondition(broker, v1alpha1.ConditionFalse, errorFetchingCatalogReason, errorFetchingCatalogMessage)
+		err:= fmt.Errorf("Error getting broker auth credentials for broker %v: %v", broker.Name, err)
+		glog.Info(err)
+		c.updateBrokerReadyCondition(broker, v1alpha1.ConditionFalse, errorFetchingCatalogReason,
```

Comment:
when we show user-defined strings (like broker.Name) I'd prefer to use %q than %s so that its quoted - its too easy for the name to be a word that actually works in the sentence and people not realize its a name.